source/devices/AM437X/linux/Release_Specific_Release_Notes.rst

```diff
@@ -150,38 +146,33 @@ Yocto
 .. rubric:: meta-ti
    :name: meta-ti
 
-| Head Commit: 59edb4e886df468edeb683355c36f06004b29925 linux-ti-staging_6.1: CI/CD Auto-Merger: cicd.kirkstone.202310191725
-| Date: 2023-10-20 08:11:47 -0500
+| Head Commit: 963140e3b5820d6ebf54a4418946a628e0fea2c6 CI/CD Auto-Merger: cicd.kirkstone.202412041235
 
 | Clone: git://git.yoctoproject.org/meta-ti
 | Branch: kirkstone
-| Release Tag: 09.01.00.001
+| Release Tag: 09.03.05
 |
 
 .. rubric:: meta-arago
    :name: meta-arago
 
-| Head Commit: a9dcc0583f44bd7a7d8780759c2091b85d280ec2 ltp-ddt: CI/CD Auto-Merger: cicd.kirkstone.202310191725
-| Date: 2023-10-20 08:11:26 -0500
+| Head Commit: f59caa5f47a625ef9eecada069ae6a74c70bcc47 CI/CD Auto-Merger: cicd.kirkstone.202412041235
 
 | Clone: git://git.yoctoproject.org/meta-arago
 | Branch: kirkstone
-| Release Tag: 09.01.00.001
+| Release Tag: 09.03.05
 |
 
-.. rubric:: meta-processor-sdk
-   :name: meta-processor-sdk
+.. rubric:: meta-tisdk
+   :name: meta-tisdk
 
-| Head Commit: 3cbe955ce03d9c25e59e3a401f378132163a3569 recipes-core: matrix: Handle systemd and sysvinit scripts separately
-| Date: 2023-10-31 03:48:05 -0500
+| Head Commit: 2ee8bead9d04afcb61234738e0d9a644395d27bf recipes-core: packagegroups: Drop SGX sources from am65xx
 
-| Clone: https://git.ti.com/git/processor-sdk/meta-processor-sdk.git
+| Clone: git://git.ti.com/ti-sdk-linux/meta-tisdk.git
```

Comment:
Do we want to use git:// instead of https:// here?

Reply:
doesnt matter , kept them to be same/consistent across devices. both urls does the job of clone .